api/File.json

```
@@ -73,6 +90,9 @@
             "ie": {
               "version_added": false
             },
+            "nodejs": {
+              "version_added": "19.2.0"
+            },
```

Looking at the Node.js docs, I think `File.{File,lastModified,name}` have also been supported since 18.13.0:

```suggestion
            "nodejs": [
              {
                "version_added": "19.2.0"
              },
              {
                "version_added": "18.13.0",
                "version_removed": "19.0.0"
              },
            ],
```